### PR TITLE
fix/secure ng bind html to prevent HTML injection

### PR DESCRIPTION
**Pull request by JeremyDec**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | yes/no
| Tickets          | Fix #MANAGER-8099
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/modules/sms/src/sms/sms/compose/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -88,8 +88,8 @@
   "sms_sms_compose_add_receivers_selected_multiple": "<strong>{{ count }}</strong> listes sélectionnées",
   "sms_sms_compose_add_receivers_lists_title": "Ajouter une liste de destinataires",
   "sms_sms_compose_add_receivers_select_lists": "Sélectionnez les fichiers contenant les destinataires :",
-  "sms_sms_compose_add_receivers_label": "{{ description }} (<small><b>{{ records }}</b> contact</small>)",
-  "sms_sms_compose_add_receivers_label_plural": "{{ description }} (<small><b>{{ records }}</b> contacts</small>)",
```
Comment (antleblanc) [inline packages/manager/modules/sms/src/sms/sms/compose/translations/Messages_fr_FR.json:92]: :+1:

File: packages/manager/modules/support/src/tickets/ticket/message/message.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
     <pre
         class="message"
-        data-ng-bind-html=":: $ctrl.message.body | htmlStringLinky:'_blank'"
+        data-ng-bind-html=":: $ctrl.message.body | linky:'_blank'"
```
Comment (antleblanc) [inline packages/manager/modules/support/src/tickets/ticket/message/message.html:17]: :+1:

File: packages/manager/modules/emailpro/src/dashboard/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -415,7 +423,8 @@
   "emailpro_resilitation_action_hosted_warning": "Vos comptes seront suspendus à échéance.",
   "emailpro_resilitation_action_partial": "Une partie des changements de la période de renouvellement sont en erreur.",
   "emailpro_resilitation_cancel_action_button": "Continuer",
-  "emailpro_resilitation_cancel_action_confirmation": "Vous êtes sur le point d'annuler la résiliation du service \"{0}\". <br />Souhaitez-vous procéder à cette opération ? <br /> Le service passera alors en renouvellement automatique.",
+  "emailpro_resilitation_cancel_action_confirmation_1": "Vous êtes sur le point d'annuler la résiliation du service \"{0}\".",
+  "emailpro_resilitation_cancel_action_confirmation_2": "Souhaitez-vous procéder à cette opération ? <br /> Le service passera alors en renouvellement automatique.",
```
Comment (antleblanc) [inline packages/manager/modules/emailpro/src/dashboard/translations/Messages_fr_FR.json:427]: **nitpick(non-blocking):** Following translation can be splitted as well.

**Review by antleblanc — APPROVED** [review]

**Review by antleblanc — DISMISSED** [review]

**Review by antleblanc — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/dedicated/server/dns/add/dedicated-server-dns-add.html [hunk 4]
```diff
@@ -50,7 +50,7 @@
                             id="domain"
                             name="domain"
                             data-ng-model="entry.domain"
-                            data-ng-pattern="/.+\..+/"
+                            data-ng-pattern="/^[^<>]+\.[^<>]+$/"
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/dedicated/server/dns/add/dedicated-server-dns-add.html:53]: No global options to forbid this ? Maybe some sort of decorator could have been helpful

Comment (JeremyDec) [reply packages/manager/apps/dedicated/client/app/dedicated/server/dns/add/dedicated-server-dns-add.html:53]: This one is specific, as it is adapted for domain input. Do you mean why not creating a decorator for other ones? (For ng-pattern="/^[^<>]+$/").
I can create it, but it implies to inject in all the concerned controllers scope, what do you think?

**Review by marie-j — APPROVED** [review]

**Pull request merged**